### PR TITLE
Add pre-migration script for account module.

### PR DESCRIPTION
**Pull request by alexis-via**
Depends on #23

### CODE REVIEW

File: addons/account/migrations/8.0.1.1/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    cr.execute(
+        """SELECT id FROM account_analytic_journal WHERE type='purchase' """)
+    res = cr.fetchone()
+    print "mig account res=", res
```
Comment (bwrsandman) [inline addons/account/migrations/8.0.1.1/pre-migration.py]: Better to use the logger than a direct print

**Comment by bwrsandman** [comment]
Whoops! Missing the `analysis_work` file.

Don't forget to update `openerp/openupgrade/doc/source/modules70-80.rst` to reflect the coverage of this module.
You may need to rebase with upstream/master

```
git remote add upstream https://github.com/OpenUpgrade/OpenUpgrade.git
git pull --rebase upstream master
```

File: addons/account/migrations/8.0.1.1/pre-migration.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.openupgrade import openupgrade
+
+
+def migrate(cr, version):
```
Comment (bguillot) [inline addons/account/migrations/8.0.1.1/pre-migration.py:26]: Can you add the decorator @openupgrade.migrate() before the method ?
Thanks!

**Comment by alexis-via** [comment]
@bwrsandman I had an analysis_work file, but I lost it in my git nightmare.

**Comment by bwrsandman** [comment]
This is the last PR set to master.
After this is closed, we should merge master into 8.0 and remove master.
@alexis-via What about the work file?

**Comment by arthru** [comment]
@bwrsandman : @alexis-via doesn't want to work on that for the moment. My branch arthru:account has that changes based on a 8.0 branch. I'm going to add the work file, and see what needs to be done with the statements.

I think we can close this PR, as I am going to create a new one for my branch.

**Comment by arthru** [comment]
Obsoleted by #74

**Pull request closed without merging**